revolv/administrator/views.py

```diff
@@ -29,6 +35,423 @@ def get_context_data(self, **kwargs):
         return context
 
 
+class AdministratorAccountingView(UserDataMixin, TemplateView):
+    """View of the administrator accounting page.
```

Comment:
Can you put a newline between the `"""` and the `View` to match the formatting of other docstrings?